docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -1,22 +1,62 @@
 ---
 title: Joining Nodes in AWS
-description: How to join nodes and proxies on EC2
+description: How to join nodes and proxies running on AWS
 h1: Joining Node and Proxies in AWS
 ---
 
 This section will explain how to set up Teleport Nodes and Proxies to join your
-Teleport cluster with the EC2 join method. The EC2 join method enables you to
-allow Teleport nodes to join your Teleport cluster from EC2 instances in your
-AWS account without sharing any secrets.
+Teleport cluster without sharing any secrets when they are running in AWS.
+
+<Admonition type="note">
+You can still use secret tokens to join the cluster, see the
+[Getting Started](../../getting-started/linux-server.mdx#add-a-node-to-the-cluster)
+guide. The following AWS join methods are meant to make deployment simpler by
+eliminating the need to distribute secrets when your Nodes or Proxies are
+running in AWS.
+</Admonition>
+
+There are two join methods available depending on your use-case.
+
+The EC2 join method:
+- is available in Teleport 7.3+ self-hosted.
+- is available to any Teleport Node or Proxy running on an EC2 instance.
+- only one Teleport Node or Proxy may use the EC2 join method per EC2 instance.
+- IAM credentials with `ec2:DescribeInstances` permissions are required on your
+  Teleport Auth server.
+- no IAM credentials are required on the Nodes or Proxies.
+
+The IAM join method:
+- is available in Teleport 8.3+ cloud or self-hosted.
+- is available to any Teleport Node or Proxy running anywhere with access to IAM
+  credentials, typically an EC2 instance with an attached IAM role.
+- no IAM credentials are required on the Teleport Auth server.
+- IAM credentials are required on the Nodes, but they do not require any
+  permissions. An IAM role with no attached policies is sufficient.
 
 ## Prerequisites
 
 - Teleport v(=teleport.version=) Open Source or Enterprise.
```

Comment:
Should we also mention Cloud here, since this supports the IAM join method?

Reply:
Yes, added Cloud